project.yaml

```diff
@@ -105,6 +111,7 @@ targets:
           exhaustive:
           - ubuntu-os-cloud:ubuntu-minimal-2404-lts-arm64
   rockylinux9:
+    os_versions: [rocky-9,rhel-9]
```

Comment:
Are we not supporting centos-9?

Reply:
Oh nvm, it does not seem like we support centos-9 now https://screenshot.googleplex.com/C74Pq9DfvZv6qeP